Automattic/woocommerce-payments
# Disputes: Categorize "fraudulent" as "unauthorized transaction"

Fixes #6372 

#### Changes proposed in this Pull Request

This PR replaces the dispute categorization of "fraudulent" with "unauthorized transaction".

There is also a misspelling of `cancelation` which was replaced with `cancellation`

### Testing instructions

1. Purchase a product using a ["dispute" test card](https://stripe.com/docs/testing#disputes) `4000000000000259` will automatically create a dispute with a reason of `fraudulent`
2. See the dispute listed under Payments > Disputes as `Transaction unauthorized`
3. Clicking through to see the dispute details should show `Transaction unauthorized`




![Screenshot 2023-06-07 at 12 37 55 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/0b01a868-2bab-4396-ae42-bcf79d97e132)
![Screenshot 2023-06-07 at 12 37 45 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/201f0834-8d0c-4c69-9155-38c1d1e1b726)



-------------------


**Post merge**
